e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-save-and-back.spec.js

```diff
@@ -54,6 +54,13 @@ describe('Insurance - Your business - Company details page - Save and go back',
 
       cy.url().should('eq', `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ALL_SECTIONS}`);
     });
+
+    it('should display the valid inputs on fields when going back to the page', () => {
+      cy.navigateToUrl(url);
+
+      yesRadioInput().first().should('be.checked');
+      yesRadioInput().eq(1).should('be.checked');
```

Comment:
Could we say "shouldrender submitted values" intead of the "valid inputs"?

Also do you think it's worth having cypress selectors for each radio e.g `hasTradingNameRadio.yes` `hasTradingAddressRadio.yes`etc instead of `first()` / `eq(1)`?

Reply:
yes, but separate ticket please as this will need a few changes

Reply:
https://ukef-dtfs.atlassian.net/browse/EMS-843